apps/settings/js/modules/bluetooth/bluetooth_adapter_manager.js

```diff
@@ -0,0 +1,113 @@
+/* global console */
+
+/**
+ * BluetoothAdapterManager:
+ *   - BluetoothAdapterManager is an Observable that wraps the platform
+ *     Bluetooth object for construct default adapter.
+ *   - It has only one observable properties: defaultAdapter
+ * BluetoothAdapterManager only update state and does not involve in any UI
+ *   logic.
+ *
+ * @module BluetoothAdapterManager
+ */
+define(function(require) {
+  'use strict';
+
+  var NavigatorBluetooth = require('modules/navigator/mozBluetooth');
+  var Observable = require('modules/mvvm/observable');
+
+  var _debug = false;
+
```

Comment:
We could define the debug function here based on the value of `_debug`. By doing this we won't have to check its value every time when calling to the debug function.